zephyr/esp32c3/src/bt/esp_bt_adapter.c

```diff
@@ -31,7 +31,6 @@
 #include <zephyr.h>
 #include <sys/printk.h>
 #include <random/rand32.h>
-#include <drivers/interrupt_controller/intc_esp32c3.h>
```

Comment:
OK, you removed it from here so that BT uses direct interrupt calls (as in WiFi), right?  
This could be a separate commit in this MR (touching BT stuff).

Reply:
Yes, it will be added back when interrupt allocator gets merged.